package.json

```diff
@@ -1111,7 +1126,7 @@
             "args": [],
             "cwd": "${workspaceRoot}",
             "stopAtEntry": false,
-            "externalConsole": false
+            "console": "internalConsole"
```

Comment:
I can go either way on this, but we might want to have 'integratedTerminal' as the default. The rationale being in the "console" launch configuration you are more likely interacting with an app that takes input. We can wait for community feedback on this too.